packages/@okta/vuepress-site/books/api-security/sanitizing/sanitize-inputs/index.md

```diff
@@ -6,13 +6,13 @@ title: Sanitize Inputs - Sanitizing Data
 
 ## Sanitize Inputs {#sanitize-inputs}
 
-Sanitizing inputs can be a good option when the input format is not strict but still somewhat predictable, such as phone numbers or other free-text fields. There are a few different ways to sanitize inputs, you could use a whitelist, a blacklist, or escape input.
+Sanitizing inputs can be a good option when the input format is not strict but still somewhat predictable, such as phone numbers or other free-text fields. There are a few different ways to sanitize inputs, you could use a allowlist, a blocklist, or escape input.
 
-### Sanitize Input Using a Whitelist
-When sanitizing data with a whitelist, only valid characters/strings matching a given pattern are kept.  For example, when validating a phone number there are multiple formats people use, US phone numbers could be written as `555-123-1245`, `(555) 123-1245`, `555.123.1245`, or a similar combination. Running any of these through a whitelist that only allows numeric characters would leave `5551231245`.
+### Sanitize Input Using a Allowlist
```

Comment:
Should this be "an Allowlist"?
cc: @damianali-okta to confirm

Reply:
Good catch.. Updated!